src/analytics/build.sbt

```diff
@@ -29,7 +29,6 @@ libraryDependencies ++= Seq(
   gtSparkTestKit,
   logging,
   scalatest,
-  apacheCommonsEmail,
```

Comment:
I attempted to figure out if any of the other dependencies here are no longer necessary with a bunch of code moving to `osmesa.apps`. I was largely unsuccessful:

1. Use [sbt-explicit-dependencies](https://github.com/cb372/sbt-explicit-dependencies) plugin to auto find unused dependencies. This plugin told me that all my deps were unused, which isn't possible. There's a few issues where people hit similar problems.
2. Use [sbt-dependency-graph](https://github.com/jrudolph/sbt-dependency-graph) plugin to list dependencies, and then look for missing ones in the list. No go, the list is massive, doesn't print well and I can't limit only to top level deps. 
3. Manually remove deps one by one. Low confidence, due to lack of test suite and things compiling that I'm not sure should? e.g. I removed the `decline` dependency but that's required by code in the `osmesa.analytics.examples` module and I was still able to compile and generate an assembly jar.
4. Use IntelliJ's [dependency analyzer](https://www.jetbrains.com/help/idea/working-with-module-dependencies.html). This ran really slowly and gave similar results to (3) and there's no search in the list to quickly look up deps. So this would require a slow manual process I don't have time for.

As a result, I only removed the dep that I know is no longer necessary as it was just added in #174 and only used in the AOIMonitor command app that was moved here to `osmesa.apps`. Perhaps we can consider a more thorough cleanup of subproject deps and build config at a later time.